motioneye/utils/dtconv.py

```diff
@@ -53,7 +53,7 @@ def pretty_date_time(date_time, tzinfo=None, short=False):
             tz += '-'
             offset = -offset
 
-        tz += '%.2d' % (offset / 3600) + ':%.2d' % ((offset % 3600) / 60)
+        tz += '%.2d' % (offset // 3600) + ':%.2d' % ((offset % 3600) // 60)
```

Comment:
OPTIONAL: Python has a nice builtin [`divmod()`](https://docs.python.org/3/library/functions.html#divmod)...
```suggestion
        div, mod = div(offset, 3600)
        tz += f"{div:02}:{mod//60:02}"
```

Reply:
For now it doesn't look so much simpler and adds inconsistency. Probably we add a `seconds_to_time` method for all such cases?

I wonder whether we can use [datetime.timezone.tzname](https://docs.python.org/3/library/datetime.html#datetime.timezone.tzname) here instead of converting from datetime to tzinfo and manually composing the string? It is in the format `UTC±HH:MM`, not `GMT±HH:MM`, which IMHO is even better.